### ISSUE TITLE
Instantiating RevoltClient throws NullReferenceException

### ISSUE THREAD

**Issue by oliverbooth** [xXBuilderBXx/RevoltSharp#6]
I have managed to narrow down the problem to a single line - the constructor call. A program as simple as this, reproduces the error:
```cs
string revoltToken = Environment.GetEnvironmentVariable("REVOLT_TOKEN")!;
var client = new RevoltClient(revoltToken, ClientMode.WebSocket);
```
The exception:
```cs
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at RevoltSharp.RevoltClient..ctor(String token, ClientMode mode, ClientConfig config)
```
I have guaranteed that `REVOLT_TOKEN` environment variable is not null, and I've even tried passing the token as a string literal. The same behaviour occurs.

**Comment by xXBuilderBXx** [comment]
Hi thanks for checking out the lib, i'll do some testing for this issue.

**Comment by xXBuilderBXx** [comment]
Ah ok i see the issue here it's using the lowercase nulled config instead of the null-coalescing Config
![image](https://user-images.githubusercontent.com/17956143/228069151-b556add8-4cb3-472e-ba00-bc4436d7b70b.png)

**Comment by xXBuilderBXx** [comment]
Fixed v4.0.5

**Comment by oliverbooth** [comment]
Wow yep even I missed that lowercase `c`.

Confirmed working in 4.0.5
